Tests/Dbal/MutableAclProviderTest.php

```diff
@@ -96,18 +97,17 @@ public function testDeleteAcl()
 
     public function testDeleteAclDeletesChildren()
     {
+        $this->expectException(AclNotFoundException::class);
```

Comment:
this should not be set here but only before the final `findAcl` call, so that it does not accept such exceptions happening earluer

Reply:
Good point, Fixed!